src/main/scala/org/bytedeco/sbt/javacpp/Plugin.scala

```diff
@@ -37,25 +37,15 @@ object Plugin extends AutoPlugin {
   private def javaCppPresetDependencies: Def.Setting[Seq[ModuleID]] = {
     import autoImport._
     libraryDependencies ++= {
-      val majorMinorJavaCppVersion = majorMinorOnly(javaCppVersion.value)
       javaCppPresetLibs.value.flatMap {
-        case (libName, libVersion) => {
-          val generic = "org.bytedeco.javacpp-presets" % libName % s"$libVersion-$majorMinorJavaCppVersion" classifier ""
+        case (libName, libVersion) =>
+          val generic = "org.bytedeco.javacpp-presets" % libName % s"$libVersion-${javaCppVersion.value}" classifier ""
```

Comment:
I think we need a smarter check here for backwards compatibility; to take into account the fact most of the existing versions depend on *not* having the patch version (see https://github.com/bytedeco/sbt-javacpp/issues/17#issuecomment-448978403)